python/docs/source/user_guide/pandas_on_spark/options.rst

```diff
@@ -290,7 +290,8 @@ compute.eager_check             True                    'compute.eager_check' se
                                                         `Series.asof`, `Series.compare`,
                                                         `FractionalExtensionOps.astype`,
                                                         `IntegralExtensionOps.astype`,
-                                                        `FractionalOps.astype`, `DecimalOps.astype`.
+                                                        `FractionalOps.astype`, `DecimalOps.astype`,
+                                                        `LocIndexer._select_cols_by_iterable`.
```

Comment:
I think maybe we should explain it instead of saying `LocIndexer._select_cols_by_iterable`. e.g.) `psdf.loc[:, iterable]`